bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/dto/onecall/Daily.java

```diff
@@ -70,6 +73,30 @@ public void setSunrise(int sunrise) {
         this.sunrise = sunrise;
     }
 
+    public int getMoonrise() {
+        return moonrise;
+    }
+
+    public void setMoonrise(int moonrise) {
```

Comment:
No need to add those setters to a DTO.
I wonder why the other fields have setters, probably because according to the JavaDoc a code generator was used.

Can you therefore please remove the added setters `setMoonrise`, `setMoonset`, `setMoon_phase`?